surfsense_backend/app/tasks/connectors_indexing_tasks.py

```
@@ -912,3 +915,192 @@ async def index_linear_issues(
+        
+        # Format dates for Discord API
+        start_date_str = start_date.isoformat()
+        end_date_str = end_date.isoformat()
+        
+        documents_indexed = 0
+        documents_skipped = 0
+        skipped_guilds = []
+        
+        try:
+            logger.info("Starting Discord bot to fetch guilds")
+            discord_client._bot_task = asyncio.create_task(discord_client.start_bot())
```

Pylint (3.3.7)

[warning] Access to a protected member _bot_task of a client class

(W0212)